.gitignore

```diff
@@ -32,6 +32,7 @@ linux/netfilter/vendor/
 __pycache__
 lottie.mjs
 qml_lint_result.txt
+.qmlls.ini
```

Comment:
I've switched to qt.6.9.0 and the qml language server drops those everywhere we have qml files. let's ignore those too,